### PR TITLE
[readme] expand install directions

### PR DESCRIPTION
**Pull request by 7596ff**
This further expands the steps that I took to install this plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,12 +11,22 @@ This plugin will download metadata about Youtube videos using a Youtube API key.
...
 1. Build plugin with following command.
-    ```sh
+    ```
     dotnet publish --configuration Release --output bin
     ```
-1. Create folder named `YoutubeMetadata` in the `plugin` directory within `data` directory.
-1. Place the resulting file from step 3 in the `plugins/YoutubeMetadata` folder created in step 4.
-1. If performed correctly you will see the plugin in the dashboard plugin page.
+1. Create folder named `YoutubeMetadata` in the `plugins` directory inside your Jellyfin data
+   directory, usually `/var/lib/jellyfin`.
```
Comment (ankenyr) [inline README.md:18]: maybe change this to specify on linux? Bonus points if you add in the OSX and Windows paths that are default.

File: README.md [hunk 2]
```diff
@@ -11,12 +11,22 @@ This plugin will download metadata about Youtube videos using a Youtube API key.
...
     dotnet publish --configuration Release --output bin
     ```
-1. Create folder named `YoutubeMetadata` in the `plugin` directory within `data` directory.
-1. Place the resulting file from step 3 in the `plugins/YoutubeMetadata` folder created in step 4.
-1. If performed correctly you will see the plugin in the dashboard plugin page.
+1. Create folder named `YoutubeMetadata` in the `plugins` directory inside your Jellyfin data
+   directory, usually `/var/lib/jellyfin`. 
+1. Place the resulting files from step 3 in the `plugins/YoutubeMetadata` folder created in step 4.
+    ```
+    $ chmod +x bin/*.dll
```
Comment (ankenyr) [inline README.md:21]: Do dll files need to have the x bit enabled? I don't believe I did this for mine.

Comment (7596ff) [reply README.md:21]: I do not know. I have just noticed every `ddl` file from dotnet applications that run on linux are `+x`. My apologies

File: README.md (hunk 1 above)
Comment (7596ff) [reply README.md:18]: Is there not a way to get this plugin into the main directory? I don't have any OSX or Windows install directions. I will generalize this part though.

Comment (ankenyr) [reply README.md]: Not sure what you mean by the first sentence. I can make it a todo for myself to find the OSX and Windows ones. Sorry to have pushed that burden onto you.

File: README.md (hunk 2 above)
Comment (ankenyr) [reply README.md]: I don't believe they need to be +x. Mine run fine and I do not have execution set. Remove that and I can merge this.

**Comment by ankenyr** [comment]
See those changes were made. This looks good how it is. Thanks!

**Pull request merged**